packages/ui/src/council/queries/council.graphql

```diff
@@ -252,6 +258,9 @@ query GetPastCouncilWorkingGroups($fromBlock: Int!, $toBlock: Int!) {
   newMissedRewardLevelReachedEvents(where: { inBlock_lte: $toBlock }, orderBy: [inBlock_DESC]) {
```

Comment:
Shouldn't `newMissedRewardLevelReachedEvents` also be restricted to the council term:
```suggestion
  newMissedRewardLevelReachedEvents(where: { inBlock_gte: $fromBlock, inBlock_lte: $toBlock }) {
```
Or am I missing something ?

Also I think `budgetSetEvents` should be removed, or is this query is used somewhere else ?

Reply:
I checked budgetSetEvents is not used in the source code. saying about newMissedRewardLevelReachedEvents, that was the original code, so i will follow you.

Reply:
I updated the #3225